### ISSUE TITLE
not finding 20180408-102900.bin or .xml for running video_face_detection

### ISSUE THREAD

**Issue by rachelglenn** [movidius/ncappzoo#370]
From the video_face_detection folder, I ran 

make deps
make data
make run

facenet: Compiling model to IR...
Checking OpenVINO environment...
Please initiate the Intel OpenVINO environment by going to the installation directory for openvino and running the setupvars.sh file in the bin folder.
Makefile:66: recipe for target 'compile_model' failed
make[1]: *** [compile_model] Error 1
make[1]: Leaving directory '/home/xilinx/workspace/ncappzoo/networks/facenet'
- Copying IRs to project folder...
cp: cannot stat '20180408-102900.bin': No such file or directory
cp: cannot stat '20180408-102900.xml': No such file or directory
Makefile:39: recipe for target 'compile_default_model' failed
make: *** [compile_default_model] Error 1

Thank you for your help in advance.

**Comment by ChristopheBrown** [comment]
I am experiencing the same issue for a different example. This is my output:

simple_classifier_py: Compiling the model to IR...
- making squeezenet IRs in squeezenet project.
make[1]: Entering directory '/home/pi/ncappzoo/ncappzoo/networks/squeezenet_v1.0'

squeezenet_v1.0: Making model zoo...
(cd ../../omz; make all;) 
make[2]: Entering directory '/home/pi/ncappzoo/ncappzoo/omz'

open_model_zoo: Checking requirements...
Checking pyyaml...
 - pyyaml already installed.


open_model_zoo: Making dependencies...

open_model_zoo: Downloading required data...
Open model zoo already exists. No need to clone.
make[2]: Leaving directory '/home/pi/ncappzoo/ncappzoo/omz'

squeezenet_v1.0: Checking zoo model files...
 - Model files already exists.

squeezenet_v1.0: Compiling model to IR...
Checking OpenVINO environment...
Intel OpenVINO environment is already set!
/bin/sh: 5: mo.py: not found
make[1]: *** [Makefile:70: compile_model] Error 127
make[1]: Leaving directory '/home/pi/ncappzoo/ncappzoo/networks/squeezenet_v1.0'
- copying IRs to project folder...
cp: cannot stat 'squeezenet1.0.bin': No such file or directory
cp: cannot stat 'squeezenet1.0.xml': No such file or directory
make: *** [Makefile:39: compile_default_model] Error 1

**Comment by RexLeeeeee** [comment]
Yes!I also have the same problem. Every examples in the ncappzoo might be wrong when I ran them in my pc. So if you have solutions, pleases reply to me and if not, maybe we could communicate with each other from the website!!

**Comment by RexLeeeeee** [comment]
F:\ncappzoo\networks\googlenet_v1>C:\MinGW\bin\make run
'\033[1;33m''\n'googlenet_v1": Downloading required data..."'\033[0m'
(cd ../../data/ilsvrc12; make all);
process_begin: CreateProcess(NULL, (cd ../../data/ilsvrc12; make all);, ...) failed.
make (e=2):
Makefile:42: recipe for target 'data' failed
make: *** [data] Error 2

F:\ncappzoo\networks\tiny_yolo_v1>C:\MinGW\bin\make run
'\033[1;33m''\n'tiny_yolo_v1": Downloading required data... No data needed."'\033[0m'
'\033[1;33m''\n'tiny_yolo_v1": Downloading prototxt file..."'\033[0m'
此时不应有 -e。
Makefile:37: recipe for target 'prototxt' failed
make: *** [prototxt] Error 255

F:\ncappzoo\apps\benchmark_ncs>C:\MinGW\bin\make run
'\033[1;33m''\n'benchmark_ncs": No data needed."'\033[0m'
'\033[1;33m''\n'benchmark_ncs": Compiling the model"'\033[0m'
此时不应有 googlenet-v1.xml。
Makefile:21: recipe for target 'compile_default_model' failed
make: *** [compile_default_model] Error 255



this is my problem.

**Comment by DoubleChuang** [comment]
cd <YOUR_WORK_SPACE>/ncappzoo/networks/facenet
source <YOUR_OPENVINO_INSTALL_DIR>/intel/openvino/bin/setupvars.sh
cd ../../networks/facenet
make compile_model
cd -
make run
